### PR TITLE
Add NDArray#shape and reshape

### PR DESCRIPTION
**Pull request by Avasil**
Not sure if `Seq[Int]` is a proper representation

### CODE REVIEW

File: project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=1.2.8
+sbt.version=1.3.3
```
Comment (PawelLipski) [inline project/build.properties]: 1.3.10? ;)

File: build.sbt [hunk 2]
```diff
@@ -4,7 +4,7 @@ import scala.sys.process._
 organization in ThisBuild := "me.shadaj"
 
 lazy val scala211Version = "2.11.12"
-lazy val scala212Version = "2.12.8"
+lazy val scala212Version = "2.12.10"
```
Comment (PawelLipski) [inline build.sbt]: 2.12.11? ;)

File: build.sbt [hunk 3]
```diff
@@ -4,7 +4,7 @@ import scala.sys.process._
 organization in ThisBuild := "me.shadaj"
 
 lazy val scala211Version = "2.11.12"
```
Comment (PawelLipski) [inline build.sbt:6]: Use `scala211Version` in `nativeSettings` as well

**Review by shadaj — APPROVED** [review]
Looks good! Thanks for the PR!

**Pull request merged**